### PR TITLE
Merge faqs

### PR DESCRIPTION
**Pull request by alvarosimon**
Includes faq pr from:
https://github.ugent.be/hpcugent/website-support-faq/pull/1

### CODE REVIEW

File: mkdocs/docs/HPC/FAQ.md [hunk 1]
```diff
@@ -1,6 +1,227 @@
-# Frequently Asked Questions 
+# Frequently Asked Questions
 
-## When will my job start?
+{% if site == gent %}
+
+New users should consult the [Introduction to HPC](https://www.ugent.be/hpc/en/training/2022/introhpcugent)
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:5]: there is no latest or other url?

File: mkdocs/docs/HPC/FAQ.md [hunk 2]
```diff
@@ -1,6 +1,227 @@
...
+and the [HPC user manual](https://www.ugent.be/hpc/en/support/documentation.htm) to get started.
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:6]: thsi is the user manual

File: mkdocs/docs/HPC/FAQ.md [hunk 3]
```diff
@@ -1,6 +1,227 @@
...
+
+The [HPC user manual](https://www.ugent.be/hpc/en/support/documentation.htm) is a great resource
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:8]: idem

File: mkdocs/docs/HPC/FAQ.md [hunk 4]
```diff
@@ -1,6 +1,227 @@
...
+Even if it works now, as more modules get installed on the HPC, your job can suddenly break.
+
+### Job script template
+
+This is a template for a job script, with commonly used parameters.
+The basic parameters should always be used. Some notes on the situational parameters:
+
+* `-l mem`: If no memory parameter is given, the job gets access to an amount of
+memory proportional to the amount of cores requested.
+See also: Job failed: SEGV Segmentation fault
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:80]: use a reference

File: mkdocs/docs/HPC/FAQ.md [hunk 5]
```diff
@@ -1,6 +1,227 @@
...
+
+Some [Job script examples](./jobscripts_examples).
+
+## Troubleshooting jobs
+
+### My modules don't work together
+
+When incompatible modules are loaded, you might encounter an error like this:
+
+Lmod has detected the following error: A different version of the 'GCC' module
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:120]: need to be put in some quotes or better yet separate file and included

File: mkdocs/docs/HPC/FAQ.md [hunk 6]
```diff
@@ -1,6 +1,227 @@
...
+loaded version of 'GCC'.
+Use 'ml spider foss' to get an overview of the available versions.
+
+Modules from the same toolchain always work together, and modules from a
+_different version of the same toolchain_ never work together.
+
+An overview of compatible toolchains can be found here:
+<https://docs.easybuild.io/en/latest/Common-toolchains.html#overview-of-common-toolchains>.
+
+See also: How do I choose the job modules?
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:132]: add reference/link

File: mkdocs/docs/HPC/FAQ.md [hunk 7]
```diff
@@ -1,6 +1,227 @@
...
+
+### My job takes longer than 72 hours
+
+The 72 hour walltime limit will not be extended. However, you can work around this barrier:
+
+* Check that all available resources are being used. See also:
+    * How many cores/nodes should I request?
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:139]: replace with links to sections?

File: mkdocs/docs/HPC/FAQ.md [hunk 8]
```diff
@@ -1,6 +1,227 @@
...
+    * My job is slow
+    * My job isn't using any GPUs
+* Use a faster [cluster](https://www.ugent.be/hpc/en/infrastructure)
+* Divide the job into more parallel processes
+* Divide the job into shorter processes, which you can submit as separate jobs
+* Use the built-in checkpointing of your software
+
+### Job failed: SEGV Segmentation fault
+
+Any error mentioning SEGV or Segmentation fault/violation has something to do with a memory error.
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:149]: need to add some quotes or soemthing around SEGV and Segmntation ...

File: mkdocs/docs/HPC/FAQ.md [hunk 9]
```diff
@@ -1,6 +1,227 @@
...
+If you weren't messing around with memory-unsafe applications or programming, your job probably hit its memory limit.
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:150]: memory limits don't trigger segfaults. those give OOM errors i think

File: mkdocs/docs/HPC/FAQ.md [hunk 10]
```diff
@@ -1,6 +1,227 @@
...
+
+When there's no memory amount specified in a job script, your job will get access to a proportional
+share of the total memory on the node: If you request a full node, all memory will be available.
+If you request 8 cores on a cluster where nodes have 2x18 cores, you will get 8/36 = 2/9
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:154]: need to add quotes or at least check this is renderend cleanly

File: mkdocs/docs/HPC/FAQ.md [hunk 11]
```diff
@@ -1,6 +1,227 @@
...
+of the total memory on the node.
+
+Try requesting a bit more memory than your proportional share, and see if that solves the issue.
+
+See also: chapter 13.2 of the [user manual](https://www.ugent.be/hpc/en/support/documentation.htm).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:159]: link to correct section

File: mkdocs/docs/HPC/FAQ.md [hunk 12]
```diff
@@ -1,6 +1,227 @@
...
+Specifically, the memory and the amount of processes you can use may present an issue.
+This is common with MATLAB compilation and Nextflow. An error caused by the login session
+limitations can look like this: `Aborted (core dumped)`.
+
+It's easy to get around these limitations: start an interactive session on one of the clusters.
+Then, you are acting as a node on that cluster instead of a login node. Notably, the slaking
+cluster will grant such a session immediately, while other clusters might make you wait a bit.
+Example command: `ml sw cluster/slaking && qsub -I -l nodes=1:ppn=8`
+
+See also: chapter 5 of the [user manual](https://www.ugent.be/hpc/en/support/documentation.htm).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:174]: replace wit link/refernece

File: mkdocs/docs/HPC/FAQ.md [hunk 13]
```diff
@@ -1,6 +1,227 @@
...
+to see which one suits your needs. Make sure that you manually switch to the GPU cluster _before_ you submit
+the job. Inside the job script, you need to explicitly request the GPUs:
+`#PBS -l nodes=1:ppn=24:gpus=2`
+
+Some software modules don't have GPU support, even when running on the GPU cluster. For example,
+when running `module avail alphafold` on the joltik cluster, you will find versions on both
+the _foss_ toolchain and the _fossCUDA_ toolchain. Of these, only the _CUDA_ versions will
+use GPU power. When in doubt, CUDA means GPU support.
+
+See also: chapter 21 of the [user manual](https://www.ugent.be/hpc/en/support/documentation.htm).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:188]: link

File: mkdocs/docs/HPC/FAQ.md [hunk 14]
```diff
@@ -1,6 +1,227 @@
...
+
+### My job runs slower than I expected
+
+There are a few possible causes why a job can perform worse than expected.
+
+Is your job using all the available cores you've requested? You can test this by increasing and
+decreasing the core amount: If the execution time stays the same, the job was not using all cores.
+Some workloads just don't scale well with more cores. If you expect the job to be very parallelizable
+and you encounter this problem, maybe you missed some settings that enable multicore execution.
+See also: How many cores/nodes should i request?
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:198]: use links/refs

File: mkdocs/docs/HPC/FAQ.md [hunk 15]
```diff
@@ -1,6 +1,227 @@
...
+
+Does your job have access to the GPUs you requested?
+See also: My job isn't using any GPUs.
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:201]: links/refs

File: mkdocs/docs/HPC/FAQ.md [hunk 16]
```diff
@@ -1,6 +1,227 @@
...
+
+Not all file locations perform the same. In particular, the \\$VSC\_HOME and \\$VSC\_DATA
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:203]: can you check if this renders nicely?

File: mkdocs/docs/HPC/FAQ.md [hunk 17]
```diff
@@ -1,6 +1,227 @@
...
+directories are, relatively, very slow to access. Your jobs should rather use the
+\\$VSC_SCRATCH directory, or other fast locations (depending on your needs), described
+in chapter 6.2 of the [user manual](https://www.ugent.be/hpc/en/support/documentation.htm).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:206]: fix link

File: mkdocs/docs/HPC/FAQ.md [hunk 18]
```diff
@@ -1,6 +1,227 @@
...
+
+Use `mympirun` in your job script instead of `mpirun`. It is a tool that makes sure everything
+gets set up correctly for the HPC infrastructure. You need to load it as a module in your
+job script: `module load vsc-mympirun`.
+
+To submit the job, use the `qsub` command rather than `sbatch`. Although both will submit a job,
+`qsub` will correctly interpret the `#PBS` parameters inside the job script. `sbatch` might not
+set the job environment up correctly for mympirun/OpenMPI.
+
+See also: chapters 7 and 26 of the [user manual](https://www.ugent.be/hpc/en/support/documentation.htm).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:222]: links

File: mkdocs/docs/HPC/FAQ.md [hunk 19]
```diff
@@ -52,18 +275,17 @@ $ <b>ls -l dataset.txt</b>
 -rwxr-x---+ 2 {{userid}} mygroup      40 Apr 12 15:00 dataset.txt
 </code></pre>
 
-For more information about `chmod` or `setfacl`, see [the section on
-chmod in chapter 3 of the Linux intro
-manual](https://hpcugent.github.io/vsc_user_docs/linux-tutorial/manipulating_files_and_directories/#changing-permissions-chmod).
+For more information about `chmod` or `setfacl`, see
+[Linux tutorial](https://hpcugent.github.io/vsc_user_docs/linux-tutorial/manipulating_files_and_directories/#changing-permissions-chmod).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:279]: the linux tutorial is a section in this page, link to it directly

File: mkdocs/docs/HPC/FAQ.md [hunk 20]
```diff
@@ -84,3 +306,63 @@ Please send an e-mail to {{hpcinfo}} that includes:
...
+
+* Manage the VO members (but can't access/remove their data on the system)
+* See how much storage each member has used, and set limits per member
+* Request additional storage for the VO
+
+One person can only be part of one VO, be it as a member or moderator.
+It's possible to leave a VO and join another one. However, it's not
+recommended to keep switching between VO's (to supervise groups, for example).
+
+See also: Chapter 6.7 of the [user manual](https://www.ugent.be/hpc/en/support/documentation.htm).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:350]: replace with link

File: mkdocs/docs/HPC/FAQ.md [hunk 21]
```diff
@@ -84,3 +306,63 @@ Please send an e-mail to {{hpcinfo}} that includes:
...
+
+### My UGent shared drives don't show up
+
+After mounting the UGent shared drives with `kinit your_email@ugent.be`,
+you might not see an entry with your username when listing `ls /UGent`.
+This is normal: try `ls /UGent/your_username` or `cd /UGent/your_username`, and you should be able to access the drives.
+Be sure to use your UGent username and not your VSC username here.
+
+See also: Chapter 6.2.5 of the [user manual](https://www.ugent.be/hpc/en/support/documentation.htm).
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:359]: link

File: mkdocs/docs/HPC/FAQ.md (hunk 9 above)
Comment (alvarosimon) [reply mkdocs/docs/HPC/FAQ.md:163]: @stdweird I did a copy of the current pr, should we remove this specific section?

**Comment by alvarosimon** [comment]
I have rendered the page, it looks okish

File: mkdocs/docs/HPC/FAQ.md [hunk 22]
```diff
@@ -1,6 +1,219 @@
-# Frequently Asked Questions 
+# Frequently Asked Questions (FAQ)
 
-## When will my job start?
+{% if site == gent %}
+
+New users should consult the [Introduction to HPC](https://www.ugent.be/hpc/en/training/2023/introhpcugent)
+and the [HPC user manual](https://docs.hpc.ugent.be) to get started.
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:6]: you must link to the welcome.md here.

File: mkdocs/docs/HPC/FAQ.md [hunk 23]
```diff
@@ -1,6 +1,219 @@
...
+
+The [HPC user manual](https://docs.hpc.ugent.be) is a great resource
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:8]: this is the same link as sentence above. can we merge the text perhaps?

File: mkdocs/docs/HPC/FAQ.md [hunk 24]
```diff
@@ -1,6 +1,219 @@
...
+for troubleshooting and looking up specifics.
+
+If you want to use software that's not yet installed on the HPC, send us a
+[software installation request](https://www.ugent.be/hpc/en/support/software-installation-request).
+
+Overview of HPC-UGent Tier-2 infrastructure:
+[https://www.ugent.be/hpc/en/infrastructure](https://www.ugent.be/hpc/en/infrastructure)
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:15]: also in a constant

File: mkdocs/docs/HPC/FAQ.md [hunk 25]
```diff
@@ -1,6 +1,219 @@
...
+
+And we run it like `mympirun ./hello.sh --output output.txt`.
+
+To our surprise, this doesn't output to the file `output.txt`, but to
+standard out! This is because `mympirun` expects the program name and
+the arguments of the program to be its last arguments. Here, the
+`--output output.txt` arguments are passed to `./hello.sh` instead of to
+`mympirun`. The correct way to run it is:
+
+<pre><code>$ <b>mympirun --output output.txt ./hello.sh</b>
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:213]: needs some shell backticks?

File: mkdocs/docs/HPC/FAQ.md [hunk 26]
```diff
@@ -52,18 +267,17 @@ $ <b>ls -l dataset.txt</b>
 -rwxr-x---+ 2 {{userid}} mygroup      40 Apr 12 15:00 dataset.txt
 </code></pre>
 
-For more information about `chmod` or `setfacl`, see [the section on
-chmod in chapter 3 of the Linux intro
-manual](https://hpcugent.github.io/vsc_user_docs/linux-tutorial/manipulating_files_and_directories/#changing-permissions-chmod).
+For more information about `chmod` or `setfacl`, see
+[Linux tutorial](../linux-tutorial/manipulating_files_and_directories/#changing-permissions-chmod).
 <!-- % \section{I no longer work for \university, can I transfer my data to another researcher working at \university}
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:272]: remove this and make an issue for it to re-add it?

File: mkdocs/docs/HPC/FAQ.md [hunk 27]
```diff
@@ -82,5 +296,72 @@ Please send an e-mail to {{hpcinfo}} that includes:
...
+The fingerprint for the RSA key sent by the remote host is
+xx:xx:xx.
+Please contact your system administrator.
+Add correct host key in /home/hostname/.ssh/known_hosts to get rid of this message.
+Offending RSA key in /var/lib/sss/pubconf/known_hosts:1
+RSA host key for user has changed and you have requested strict checking.
+Host key verification failed.
+```
+
+Please follow the instructions at <https://www.ugent.be/hpc/en/infrastructure/migration\_to\_rhel8>
```
Comment (stdweird) [inline mkdocs/docs/HPC/FAQ.md:326]: actually, that page also has to be moved here ;)

Comment (stdweird) [reply mkdocs/docs/HPC/FAQ.md:326]: (well, not in the faq, but in eg the only/gent like the donphan-gallade page

**Comment by alvarosimon** [comment]
@stdweird I have included the md changes requests

**Pull request merged**